File: hw_06/Java/java_oopwork06/src/geometry/Circle.java
public class Circle {

    private double radius;

    public Circle(double radius) {
        this.radius = radius;
    }

    public Circle() {
        this(1.0);
    }

    public double getRadius() {
        return radius;
    }

    public void setRadius(double radius) {
        this.radius = radius;
    }

    public double area() {
        return Math.PI * Math.pow(radius, 2);
    }

    public void showCirInfo() {
        System.out.println("\nCircle:");
        System.out.printf("Radius = %.2f%n", radius);
        System.out.printf("Area = %.2f%n", area());
    }
}

File: hw_06/CS/cs_oopwork06/cs_oopwork06/geometry/Circle.cs
using System;
using System.Collections.Generic;
using System.IO.Compression;
using System.Linq;
using System.Threading.Tasks;

namespace cs_oopwork06
{
    public class Circle
    {
        private double radius;

        public double GetRadius()
        {
            return radius;
        }

        public void SetRadius(double radius)
        {
            this.radius = radius;
        }

        public Circle(double radius)
        {
            this.radius = radius;
        }

        public Circle(): this(1.0){}

        public double area()
        {
            return Math.PI * Math.Pow(radius, 2);
        }

        public void cirInfo()
        {
            System.Console.WriteLine($"Circle: Radius = {radius:N2}, Area = {area():N2}");
        }
    }

}
